velox/connectors/tpch/tests/TpchConnectorTest.cpp

```diff
@@ -97,11 +97,11 @@ TEST_F(TpchConnectorTest, simple) {
       makeFlatVector<int64_t>({0, 1, 1, 1, 4}),
       // n_comment
       makeFlatVector<StringView>({
-          " haggle. carefully final deposits detect slyly agai",
-          "al foxes promise slyly according to the regular accounts. bold requests alon",
-          "y alongside of the pending deposits. carefully special packages are about the ironic forges. slyly special ",
-          "eas hang ironic, silent packages. slyly regular packages are furiously over the tithes. fluffily bold",
-          "y above the carefully unusual theodolites. final dugouts are quickly across the furiously regular d",
+          "furiously regular requests. platelets affix furious",
```

Comment:
Can we add a test for 300MB as well?
We can specify the TableHandle. I also think we need to extend TableHandle for Prestissimo? 
```
.tableHandle(std::make_shared<TpchTableHandle>(
                      kTpchConnectorId, Table::TBL_SUPPLIER, scaleFactor))
```